### DISCUSSION TITLE
Support for apple silicon

### DISCUSSION THREAD

**Discussion by adviner** [vladmandic/face-api#85] [General]
I'm wondering if your project supports apple silicon?

**Comment by vladmandic** [comment]
for browser environments, it is supported since safari provides webgl implemementation   
for node environments, i don't provide support out-of-the-box since there is no `@tensorflow/tfjs-node` for **m1** architecture  

there are ways to manually compile it, but its really complex  
tfjs team is preparing `tfjs-node` for m1 and it should be released in the next version of tfjs  
as soon as it is released, i'll support it

**Reply by adviner** [reply]
That's awesome.  Thanks
